nilearn/plotting/surf_plotting.py

```diff
@@ -423,17 +426,29 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
-def _check_hemisphere(hemisphere):
+def _check_hemispheres(hemispheres):
     """Checks whether the hemisphere in plot_img_on_surf is correct.
 
-    hemisphere: :obj:`str`
+    hemisphere: :obj:`list`
```

Comment:
Nilearn tends to go with simpler styles. One of the following for example: 
```
hemisphere: List[strings]
hemisphere: list of strings
hemisphere: list
```
Checking out existing function's docstrings will be helpful.